logo_usage_guidelines.md

```diff
@@ -1,18 +0,0 @@
-# Kubernetes Branding Guidelines
```

Comment:
I think I did mv it, I don;t know why it's not showing as such, but it's not that important.